src/main/cpp/events/baseaudioevent.h

```diff
@@ -63,6 +63,13 @@ class BaseAudioEvent
         virtual void mixBuffer( AudioBuffer* outputBuffer, int bufferPosition, int minBufferPosition,
                                 int maxBufferPosition, bool loopStarted, int loopOffset, bool useChannelRange );
 
+        /**
+         * an AudioEvent can also synthesize audio "live" which actually implies the event
+         * is not sequenced, but triggered by a manual "note on" / "note off" action.
+         * This method is called during the write cycle on each buffer update of the AudioRenderer
+         */
+        virtual void mixBuffer( AudioBuffer* outputBuffer );
+
```

Comment:
Old name _synthesize_ was confusing, basically this is the same as _mixBuffer_, just overload the method to imply it is for non-sequenced event usage.